extension/chrome/elements/attachment.ts

```diff
@@ -273,16 +329,30 @@ export class AttachmentDownloadView extends View {
       if (!result.filename || ['msg.txt', 'null'].includes(result.filename)) {
         result.filename = this.attachment.name;
       }
-      Browser.saveToDownloads(new Attachment({ name: result.filename, type: this.attachment.type, data: result.content }));
+      Browser.saveToDownloads(
+        new Attachment({ name: result.filename, type: this.attachment.type, data: result.content })
+      );
     } else if (result.error.type === DecryptErrTypes.needPassphrase) {
-      BrowserMsg.send.passphraseDialog(this.parentTabId, { type: 'attachment', longids: result.longids.needPassphrase });
-      if (! await PassphraseStore.waitUntilPassphraseChanged(this.acctEmail, result.longids.needPassphrase, 1000, this.ppChangedPromiseCancellation)) {
+      BrowserMsg.send.passphraseDialog(this.parentTabId, {
+        type: 'attachment',
+        longids: result.longids.needPassphrase
+      });
+      if (
+        !(await PassphraseStore.waitUntilPassphraseChanged(
```

Comment:
and here. There will be a TON of places like this. If you actually do prefer `!(await something)` than that is ok with me.